dataland-frontend/tests/e2e/fixtures/lksg/LKSGdataFixtures.ts

```diff
@@ -0,0 +1,175 @@
+import { faker } from "@faker-js/faker";
+import { LksgData, ProductionSite } from "../../../../build/clients/backend";
```

Comment:
Maybe use import { xyz } from @abc instead of import { xyz} from "../../../..somewhere"

Reply:
Repaired.